packages/primitives-core/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "devDependencies": {
     "@emotion/react": "11.9.3",
+    "@types/css-to-react-native": "^3.0.0",
```

Comment:
could you check if this package gets referenced anywhere in the generated `.d.ts`?

Reply:
It does not get referenced in the generated `.d.ts`.